src/app/components/MatchItem/MatchItem.module.css

```diff
@@ -0,0 +1,12 @@
+.container {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: space-between;
```

Comment:
You use this three properties two times in this pr, would it be efficient to put this into a global css file and attach both classes to the html elements?